Update Vivotek camera component

HarlemSquirrel opened the pull request:
Load model name to instance attribute

## Description:

- Load model name to instance attribute
- Update [libpyvivotek to 0.2.2](https://pypi.org/project/libpyvivotek/0.2.2/)
    - Retrieve camera model name anonymously
    - Raise a more helpful error for invalid creds

**Related issue (if applicable):** fixes #26607

## Checklist:
  - [x] The code change is tested and works locally.
  - [ ] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [ ] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [ ] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [ ] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [ ] Untested files have been added to `.coveragerc`.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

HarlemSquirrel committed:
Update Vivotek camera component

Load model name to instance attribute

MartinHjelmare reviewed (changes requested):
Call `add_entities` with second argument `True` to make sure `update` is called during entity addition.
```py
add_entities(entities, True)
```

HarlemSquirrel committed:
Update Vivotek camera component

Ensure `update` is called when the entity is added.

HarlemSquirrel committed:
Update libpyvivotek to 0.2.2

https://pypi.org/project/libpyvivotek/0.2.2/
- Retrieve camera model name anonymously
- Raise a more helpful error for invalid creds

MartinHjelmare reviewed (approved):
Good!
